tests/system/action/user/test_update.py

```diff
@@ -75,10 +82,24 @@ def test_update_template_fields(self) -> None:
                 "structure_level_$": {1: "level_1", 2: "level_2"},
                 "about_me_$": {1: "<p>about</p><iframe></iframe>"},
                 "vote_weight_$": {1: "1.000000", 2: "2.333333"},
+                "committee_$_management_level": {
+                    1: None,
```

Comment:
Just a clairification: When this line were missing, the user would have CML for committees 1 and 2, but `user/committee_ids` would be `[2]` after the update?

Reply:
yes